packages/calypso-e2e/src/lib/pages/themes-detail-page.ts

```diff
@@ -23,4 +33,40 @@ export class ThemesDetailPage extends BaseContainer {
 		await this.page.click( selectors.demoPane );
 		await PreviewComponent.Expect( this.page );
 	}
+
+	/**
+	 * Activates the theme.
+	 *
+	 * If the optional parameter `keepModal` is set to true, the Thanks message modal will not
+	 * be dismissed.
+	 *
+	 * @param {boolean} [keepModal] Optional parameter that if set to true will keep the thanks modal open. Defaults to false.
+	 * @returns {Promise<void>} No return value.
+	 */
+	async activate( { keepModal = false }: { keepModal?: boolean } = {} ): Promise< void > {
+		await this.page.click( selectors.activateDesignButton );
+		await this.page.waitForSelector( selectors.activateModal );
+		await this.page.click( selectors.activateModalButton );
+		await this.page.waitForSelector( selectors.thanksMessage );
+		if ( ! keepModal ) {
+			await this.page.keyboard.press( 'Escape' );
+		}
+	}
+
+	/**
+	 * Clicks on the Customize site button once the theme has been applied
+	 * from the details page.
+	 *
+	 * This method will expect and return a new page that will be launched in the current
+	 * browser context.
+	 *
+	 * @returns {Promise<Page} New tab/popup page.
+	 */
+	async customizeSite(): Promise< Page > {
+		const [ popup ] = await Promise.all( [
+			this.page.waitForEvent( 'popup' ),
+			this.page.click( selectors.customizeDesignButton ),
```

Comment:
Clicking on the `Customize site` span at top of the page **_after_** the theme has been applied will bring up the customizer in a popup.